modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -12,13 +12,13 @@ def initialize(user)
           @user = user
         end
 
-        def get_appointments(start_date:, end_date:, include_pending:, pagination_params: {})
+        def get_appointments(start_date:, end_date:, include_pending:, include_claims: false, pagination_params: {})
           statuses = include_pending ? VAOS_STATUSES : VAOS_STATUSES.excluding('proposed')
 
           # VAOS V2 appointments service accepts pagination params but either it formats them incorrectly
           # or the upstream service does not use them.
           response = vaos_v2_appointments_service.get_appointments(start_date, end_date, statuses.join(','),
-                                                                   pagination_params, include_params)
+                                                                   pagination_params, include_params(include_claims))
```

Comment:
Would it make sense to change the call to `include_params` to instead be a hash?

defined above (say, line 17): 
```ruby
include_params = {
  clinics: true,
  facilities: true,
  travel_pay_claims: include_claims
}
```

then used like:
```ruby
response = vaos_v2_appointments_service.get_appointments(start_date, end_date, statuses.join(','),
                                                                   pagination_params, include_params)
```
and then you don't have to complicate the very simple method by passing a param.